bokehjs/src/lib/models/glyphs/circle.ts

```diff
@@ -161,22 +160,25 @@ export class CircleView extends XYGlyphView {
     const y = this.renderer.yscale.invert(sy)
     const {hit_dilation} = this.model
 
-    let x0, x1, y0, y1
-    if (this.use_radius && this.model.properties.radius.units == "data") {
-      x0 = x - this.max_radius*hit_dilation
-      x1 = x + this.max_radius*hit_dilation
-
-      y0 = y - this.max_radius*hit_dilation
-      y1 = y + this.max_radius*hit_dilation
-    } else {
-      const sx0 = sx - this.max_size*hit_dilation
-      const sx1 = sx + this.max_size*hit_dilation
-      ;[x0, x1] = this.renderer.xscale.r_invert(sx0, sx1)
-
-      const sy0 = sy - this.max_size*hit_dilation
-      const sy1 = sy + this.max_size*hit_dilation
-      ;[y0, y1] = this.renderer.yscale.r_invert(sy0, sy1)
-    }
+    const [x0, x1, y0, y1] = (() => {
+      if (this.use_radius && this.model.properties.radius.units == "data") {
+        const dr = this.max_radius*hit_dilation
+        const x0 = x - dr
+        const x1 = x + dr
+        const y0 = y - dr
+        const y1 = y + dr
+        return [x0, x1, y0, y1]
```

Comment:
just a suggestion, but these expressions are simple/small enough that this seems preferable:

```suggestion
        return [x - dr, x + dr, y - dr, y + dr]
```
(and similar in the calls to `r_invert` below)